### PR TITLE
t_33_postgres

### PR DESCRIPTION
**Pull request by Olexbaiko**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -4,10 +4,16 @@
 # Define the application directory
 import os
 BASE_DIR = os.path.abspath(os.path.dirname(__file__))  
+#identify variable db_name and user pass
+
+db_username = os.getenv("READER_USERNAME")
```
Comment (crchemist) [inline config.py:9]: please provide default values for db_username, db_password and db_name:

```
db_username = os.getenv("PSQL_USERNAME", 'cit')
```

File: config.py [hunk 2]
```diff
@@ -4,10 +4,16 @@
...
+db_password = os.getenv("READER_PASSWORD")
+db_name = os.getenv("DATABASE_NAME") or 'cit'
 
 # Define the database - we are working with
-# SQLite for this example
-SQLALCHEMY_DATABASE_URI = 'sqlite:///' + os.path.join(BASE_DIR, 'cit.db')
+# SQLite change for postpreSQL
+from sqlalchemy import create_engine
```
Comment (crchemist) [inline config.py:15]: you don't need any imports.  pls remove this line.

File: config.py [hunk 3]
```diff
@@ -4,10 +4,16 @@
...
+engine = create_engine('postgresql+psycopg2://localhost/cit', isolation_level="AUTOCOMMIT")
```
Comment (crchemist) [inline config.py:16]: connection string is stored in SQLALCHEMY_DATABASE_URI and is used by init_db.py and run.py

File: config.py (hunk 1 above)
Comment (Olexbaiko) [inline config.py:9]: I see
So, I'll correct it as fast as possible
28 квіт. 2015 18:27, користувач "Mykola Kharechko" notifications@github.com
написав:

**Pull request closed without merging**